checkov/terraform/module_loading/content.py

```diff
@@ -3,8 +3,9 @@
 
 
 class ModuleContent(object):
-    def __init__(self, dir: Optional[Union[tempfile.TemporaryDirectory, str]]) -> None:
-        self.dir = dir
+    def __init__(self, dir: Optional[Union[tempfile.TemporaryDirectory, str]], next_url=None, exception=None) -> None:
+        self.dir = dir.replace('//', '/') if dir else dir
```

Comment:
@rotemavni `if dir else dir`? In which case is dir None? If that's the case - let the `else` case be None to be clearer please